cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceConfig.java

```diff
@@ -28,14 +28,22 @@
   @SerializedName("scheduler.queue.name")
   private final String schedulerQueueName;
 
-  public NamespaceConfig(String schedulerQueueName) {
+  @SerializedName("hbase.namespace")
```

Comment:
this will be removed once HDFS mapping PR is merged. i had to use this to make the code compile.